suite-native/message-system/src/selectors.ts

```diff
@@ -0,0 +1,30 @@
+import { memoize } from 'proxy-memoize';
+
+import { MessageSystemRootState } from '@suite-common/message-system';
+
+export const selectAllActiveMessageIds = memoize((state: MessageSystemRootState) => {
+    const { validMessages, dismissedMessages } = state.messageSystem;
+
+    // At the moment, in suite-native/message-system we are not distinguishing between different message
+    // categories. For that reason all categories are merged to a one single array.
```

Comment:
I don't think it's a good idea to merge all the categories. Better to use/filter just banner category and ignore the rest. Then you can implement other category for later version o the app and start using it just from that version without breaking older versions. With message-system always think about the backward compatibility.

(But if you keep it as it is, in fact it would be still possible to target later messages by app version, it's just unnecessary complication.)

Reply:
After discussion, only the `banner` message category is used. So no need for merging categories together. Fixup https://github.com/trezor/trezor-suite/commit/1fc0085db992732ed544631274226788618721b2.